src/main/scala/cosas/ops/types/SerializeDenotations.scala

```diff
@@ -0,0 +1,44 @@
+package ohnosequences.cosas.ops.types
+
+import ohnosequences.cosas._, types._, typeSets._, fns._
+
+// TODO msg not found
+trait SerializeDenotations[Denotations <: AnyTypeSet, V]
+extends Fn2[Denotations, Map[String,V]] with
```

Comment:
if you want to serialize the values of two records to a map, this way you will get an error; if you do it the other way map ++ will happily overwrite your first map